cluster/src/main/java/org/apache/iotdb/cluster/query/ClusterPlanExecutor.java

```diff
@@ -707,5 +709,24 @@ public void delete(Path path, long timestamp) throws QueryProcessException {
     }
   }
 
+  @Override
+  public void delete(DeletePlan deletePlan) throws QueryProcessException {
+    try {
+      Set<String> existingPaths = new HashSet<>();
+      for (Path p : deletePlan.getPaths()) {
+        existingPaths.addAll(getPathsName(p.getFullPath()));
+      }
+      if (existingPaths.isEmpty()) {
+        logger.info("TimeSeries does not exist and its data cannot be deleted");
+        return;
```

Comment:
This change surely can abandon unnecessary exception, but what if the user deletes a time series which really doesn't exist? I think distributed IoTDB should also throw an exception to user as same as stand-alone IoTDB. Maybe you should add a judgement in `processNonPartitionedDataPlan` after getting all paths.

Reply:
Thank you for your suggestions :)

Reply:
Agreed, information returned to the user should be consistent with the stand-alone version as much as possible.